testsuite/features/secondary/allcli_system_group.feature

```diff
@@ -8,31 +8,51 @@ Feature: Manage a group of systems
   Scenario: Log in as org admin user
     Given I am authorized
 
+  Scenario: Pre-require: enable old packages to fake a possible installation and check that were created
+    When I enable repository "test_repo_rpm_pool" on this "sle_minion"
+    And I refresh the metadata for "sle_minion"
+    And I install old package "andromeda-dummy-1.0" on this "sle_minion"
+    And I install old package "virgo-dummy-1.0" on this "sle_minion"
+    And I follow the left menu "Admin > Task Schedules"
+    And I follow "errata-cache-default"
+    And I follow "errata-cache-bunch"
+    And I click on "Single Run Schedule"
+    Then I should see a "bunch was scheduled" text
+    And I wait until the table contains "FINISHED" or "SKIPPED" followed by "FINISHED" in its first rows
+    When I follow the left menu "Patches > Patch List > Relevant"
+    Then I should see an update in the list
+    When I wait until I see "andromeda-dummy" text, refreshing the page
```

Comment:
```suggestion
    Then I wait until I see "andromeda-dummy" text, refreshing the page
```

Reply:
To wait is an action, not a test, but okay, there seems to be no team agreement on that